src/pages/permissions/PermissionIdentities.tsx

```diff
@@ -0,0 +1,255 @@
+          role: "cell",
+          className: "u-align--right",
+          "aria-label": "Groups for this identity",
+        },
+      ],
+      sortData: {
+        id: identity.id,
+        name: identity.name.toLowerCase(),
+        authentication_method: identity.authentication_method,
+        type: identity.type,
+        groups: identity.groups?.length || 0,
+      },
+    };
+  });
+
+  const { rows: sortedRows, updateSort } = useSortTableData({ rows });
+
+  const selectedIdentities = identities.filter((identity) =>
+    selectedIdentityNames.includes(identity.name),
+  );
+
+  // NOTE: tls user group membership cannot be modified, this will be supported in the future
+  const nonTlsUsers = identities.filter((identity) => {
+    const isTlsIdentity = identity.authentication_method === "tls";
+    return !isTlsIdentity;
+  });
+
+  if (isLoading) {
+    return <Loader text="Loading identities" />;
+  }
+
+  return (
+    <>
+      <CustomLayout
+        contentClassName="u-no-padding--bottom"
+        header={
+          <PageHeader>
+            <PageHeader.Left>
+              <PageHeader.Title>
+                <HelpLink
+                  href={`${docBaseLink}/explanation/authorization`}
+                  title="Learn more about permissions"
+                >
+                  Identities
+                </HelpLink>
+              </PageHeader.Title>
+              {!selectedIdentityNames.length && !panelParams.panel && (
+                <PageHeader.Search>
+                  <PermissionIdentitiesFilter />
+                </PageHeader.Search>
+              )}
+              {!!selectedIdentityNames.length && (
+                <EditIdentityGroupsBtn
+                  identities={selectedIdentities}
+                  className="u-no-margin--bottom"
+                />
+              )}
+            </PageHeader.Left>
+          </PageHeader>
+        }
+      >
+        <NotificationRow />
+        <Row className="permission-identities">
+          <ScrollableTable
+            dependencies={[identities]}
+            tableId="identities-table"
+            belowIds={["status-bar"]}
+          >
+            <TablePagination
+              data={sortedRows}
+              id="pagination"
+              itemName="identity"
+              className="u-no-margin--top"
+              aria-label="Table pagination control"
+              description={
+                selectedIdentityNames.length > 0 && !panelParams.panel ? (
+                  <SelectedTableNotification
+                    totalCount={nonTlsUsers.length ?? 0}
+                    itemName="OIDC identity"
+                    selectedNames={selectedIdentityNames}
+                    setSelectedNames={setSelectedIdentityNames}
+                    filteredNames={nonTlsUsers.map((item) => item.name)}
+                  />
+                ) : (
+                  `Showing all ${pluralize("identity", sortedRows.length)}`
```

Comment:
Yes agreed, I think when the groups panel is open, we should show only the selected identities. Then the copy can be 'Showing selected identity/identities'. wdyt?

Reply:
I think this is the main identities list, unrelated to the side panel. The problem is if there is just 1 identity available, the copy will say "Showing all identity", which seems off. On instances we say "Showing 1 of 1 instance", which might work here as well.

Reply:
Yes I am referring to the identities list, when the groups panel is open, I think maybe we only show the selected identities? Let's say if there are a lot of identities, the user filtered for some identities and selected them, when the groups panel is open then it could be good to have only the selected identities showing for easier reference

Reply:
We could do that, it might be a good improvement. Maybe let's keep it as is for now to avoid feature creep. The changeset for this PR is already too big. We can note the idea as a possible improvement in a new JIRA and have it as part of a follow-up PR.

Reply:
I am afraid it might be a confusing experience, hence I am a bit hesitant.

Reply:
Maybe I can demo it to you quick later then decide? If it's confusing then we can just update the copy which is a quick change anyway :slightly_smiling_face: